django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
```

Comment:
Why would you not want to protect against a naked read() that will gobble the whole request into memory, no matter the content-type?